packages/web-twig/src/Resources/components/Pagination/PaginationWrapper.twig

```diff
@@ -0,0 +1,15 @@
+{# API #}
+{%- set props = props | default([]) -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _classNames = [ _styleProps.className ] -%}
+
+<nav
+    {{ mainProps(props) }}
+    {{ styleProp(_styleProps) }}
+    {{ classProp(_classNames) }}
```

Comment:
We can use this directly, can't we?

Reply:
Generally, we can, but this adds two features out of the box. 
1. it makes adding new classes easy
2. it checks if there are any values and outputs the class attribute only if so

And it was easier for me to implement 😁 
If you insist on direct usage, I can di refactoring.